templates/base.html

```diff
@@ -0,0 +1,44 @@
+{% load compress umap_tags %}
+{% load static from staticfiles %}
+<!DOCTYPE html>
+<html>
+<head>
+    <title>{% block head_title %}Cartes Data Gouv{% endblock %}</title>
+    <link rel="shortcut icon" href="{% static "favicon.ico" %}" />
+    <meta charset="utf-8">
+    {% compress css %}
+    <!-- Included CSS Files (Compressed) -->
+    <link rel="stylesheet" href="{{ STATIC_URL }}storage/contrib/css/storage.ui.default.css">
+    {% endcompress css %}
+    <!-- IE Fix for HTML5 Tags -->
+    <!--[if lt IE 9]>
+    <script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script>
+    <![endif]-->
+
+    {% block extra_head %}
+    {% endblock extra_head %}
+
+    {% compress js %}
+      <script src="{{ STATIC_URL }}umap/js/autocomplete.js"></script>
```

Comment:
We may need to integrate some css for the autocomplete (which was I think in the default umap.css, but may be moved to an another file to prevent copy-pastasciuting)